plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardRuntimeChangeListener.java

```diff
@@ -107,7 +106,6 @@ public void handleEvent(IFacetedProjectEvent event) {
     }
 
     // Add the App Engine facet
-    IProject project = facetedProject.getProject();
```

Comment:
`FacetedProject#getProject()` is just a getter, and a `FacetedProject` is created from an `IProject`.  So this is free of side-effects.